app/controllers/api/v1/users_controller.rb

```diff
@@ -1,16 +1,17 @@
 module Api::V1
   class UsersController < ApiBaseController
-    before_action :set_user, only: [:permissions, :show, :update, :destroy]
+    before_action :set_user, only: [:show, :update, :destroy]
 
     # GET /v1/users
     def index
       @users = User.all
       render json: @users, include: 'media', status: :ok
     end
 
-    # GET /v1/users/{id}/permissions
+    # GET /v1/users/permissions
```

Comment:
I'm wondering if this path would make more sense as `/v1/users/current/permissions`. What do you think?

Reply:
Yeah, I like that. More clear for sure. Pushing now.